e2e/embedding-sdk-host-apps/create-react-app-5-host-app/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+rm -rf build
+
+npm ci --install-links
+npm audit --audit-level=low
```

Comment:
I guess this fails the build if there is some security vulns in the host app's dependencies?

Reply:
Yes, and strangely, from today I got 3 moderate errors that I'm not sure how to fix. This may be a blocker for this PR if i won't be able to resolve them.